docs/reference/modules/coordination.asciidoc

```diff
@@ -0,0 +1,497 @@
+looks like this:
+
+[source,js]
+--------------------------------------------------
+{
+  "master_nodes":[
+    {"id":"USpTGYaBSIKbgSUJR2Z9lg","name":"master-a"},
+    {"id":"gSUJR2Z9lgUSpTGYaBSIKb","name":"master-b"},
+    {"id":"2Z9lgUSpTgSUYaBSIKbJRG","name":"master-c"}
+  ]
+}
+--------------------------------------------------
+
+To bootstrap a cluster, the administrator must identify a suitable set of
+master-eligible nodes, construct a bootstrap warrant, and pass the warrant to
+the `POST /_cluster/bootstrap` API:
+
+[source,js]
+--------------------------------------------------
+# send the bootstrap warrant back to the cluster
+POST /_cluster/bootstrap
+{
+  "master_nodes":[
+    {"id":"USpTGYaBSIKbgSUJR2Z9lg","name":"master-a"},
+    {"id":"gSUJR2Z9lgUSpTGYaBSIKb","name":"master-b"},
+    {"id":"2Z9lgUSpTgSUYaBSIKbJRG","name":"master-c"}
+  ]
+}
+--------------------------------------------------
+// CONSOLE
+
+This only needs to occur once, on a single master-eligible node in the cluster,
+but for robustness it is safe to repeatedly call `POST /_cluster/bootstrap`,
+and to call it on different nodes concurrently. However **it is vitally
+important** to use the same bootstrap warrant in each call.
+
+WARNING: You must pass the same bootstrap warrant to each call to `POST
+/_cluster/bootstrap` in order to be sure that only a single cluster forms
+during bootstrapping and therefore to avoid the risk of data loss.
+
+The simplest and safest way to construct a bootstrap warrant is to use the `GET
+/_cluster/bootstrap` API:
+
+[source,js]
+--------------------------------------------------
+# Immediately return a bootstrap warrant based on the nodes discovered so far
+GET /_cluster/bootstrap
+# Wait until the node has discovered at least 3 nodes, or 60 seconds has elapsed,
+# and then return the resulting bootstrap warrant
+GET /_cluster/bootstrap?wait_for_nodes=3&timeout=60s
+--------------------------------------------------
+// CONSOLE
+
+This API returns a properly-constructed bootstrap warrant that is ready to pass
+to the `POST /_cluster/bootstrap` API.  It includes all of the master-eligible
+nodes that the handling node has discovered via the gossip-based discovery
+protocol, and returns an error if fewer nodes have been discovered than
+expected.
+
+It is also possible to construct a bootstrap warrant manually and to specify
+the initial set of nodes in terms of their names alone, rather than including
+their IDs too:
+
+[source,js]
+--------------------------------------------------
+# send the bootstrap warrant back to the cluster
+POST /_cluster/bootstrap
+{
+  "master_nodes":[
+    {"name":"master-a"},
+    {"name":"master-b"},
+    {"name":"master-c"}
+  ]
+}
+--------------------------------------------------
+// CONSOLE
+
+It is safer to include the node IDs, in case two nodes are accidentally started
+with the same name.
+
+This process is implemented in the `elasticsearch-bootstrap-cluster`
```

Comment:
*This process*
It's not clear what "this" is exactly referring to here, especially given the preceding sentence.

Maybe give this another sub-heading and say "A simpler way for bootstrapping a cluster ..."